.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,50 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  release:
+    types: [published]
+  workflow_dispatch:
+
+name: 🌐 Build site
+
+permissions: read-all
+
+jobs:
+  pkgdown:
+    runs-on: ubuntu-latest
+    # Only restrict concurrency for non-PR jobs
+    concurrency:
+      group: pkgdown-${{ github.event_name != 'pull_request' || github.run_id }}
+    env:
+      GITHUB_PAT: ${{ secrets.TEST_GITHUB_PAT }}
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install R dependencies
+        run: |
+          Rscript -e 'install.packages(c("pkgdown", "remotes"))'
+          Rscript -e 'remotes::install_github("r-world-devs/GitAI")'
```

Comment:
Also  would that work without pak installation of elmer?

Reply:
Do you mean precise branch for `GitAI`? All changes are now merged to main. The new page will be published now after every merge. Why do you think we should need precise branch?
The job works now without installing `elmer`. In future it may be needed when we build more vignettes.

Reply:
Thanks, let's leave it as is then. I skipped the part that docs is built only on main, in this case it's fine.